packages/rum-core/src/domain/view/viewMetrics/trackInteractionToNextPaint.ts

```diff
@@ -74,8 +99,6 @@ export function trackInteractionToNextPaint(
 
   return {
     getInteractionToNextPaint: (): InteractionToNextPaint | undefined => {
-      // If no INP duration where captured because of the performanceObserver 40ms threshold
-      // but the view interaction count > 0 then report 0
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** ‏this comment was removed, is it intended?